### PR TITLE
chore: fix form ids to 4 for 7540 in tests & scripts [SUP-7526]

### PR DESCRIPTION
**Pull request by sujithsomraaj**

### CODE REVIEW

File: test/utils/BaseSetup.sol [hunk 1]
```diff
@@ -195,8 +195,8 @@ abstract contract BaseSetup is StdInvariant, Test {
     /// @notice tUSD is a test token on sepolia
     string[] public UNDERLYING_TOKENS = ["DAI", "USDC", "WETH", "ezETH", "wstETH", "sUSDe", "USDe", "tUSD"];
 
-    /// @dev 1 = ERC4626Form, 4 = ERC5115, 5 is ERC7540
-    uint32[] public FORM_IMPLEMENTATION_IDS = [uint32(1), uint32(4), uint32(5)];
+    /// @dev 1 = ERC4626Form, 4 = ERC5115,  3 is ERC7540,
```
Comment (0xTimepunk) [inline test/utils/BaseSetup.sol:198]: Change ERC5115 to id 3 to match what is on scripts. That way ERC7540 can be id 4 everywhere, including scripts

File: test/utils/BaseSetup.sol [hunk 2]
```diff
@@ -1058,17 +1057,13 @@ abstract contract BaseSetup is StdInvariant, Test {
             contracts[vars.chainId][bytes32(bytes("ERC5115Form"))] = vars.erc5115form;
 
             //  ERC7540 Form
-            /// @dev TODO: change id of form to 2
-            vars.erc7540form = address(new ERC7540Form{ salt: salt }(vars.superRegistry, 5));
-
+            vars.erc7540form = address(new ERC7540Form{ salt: salt }(vars.superRegistry, 3));
```
Comment (0xTimepunk) [inline test/utils/BaseSetup.sol:1060]: Id 4 here

File: test/utils/BaseSetup.sol [hunk 3]
```diff
@@ -1517,7 +1512,7 @@ abstract contract BaseSetup is StdInvariant, Test {
                                 contracts[chainIds[i]][bytes32(bytes("SuperformFactory"))]
                             ).createSuperform(FORM_IMPLEMENTATION_IDS[j], vault);
 
-                            if (FORM_IMPLEMENTATION_IDS[j] == 5) {
+                            if (FORM_IMPLEMENTATION_IDS[j] == 3) {
```
Comment (0xTimepunk) [inline test/utils/BaseSetup.sol:1515]: Id 4 here

File: test/utils/BaseSetup.sol [hunk 4]
```diff
@@ -2168,19 +2164,19 @@ abstract contract BaseSetup is StdInvariant, Test {
...
-        vaultBytecodes2[5].vaultKinds.push("ERC7540AsyncDepositMock");
-        vaultBytecodes2[5].vaultBytecode.push(type(ERC7540AsyncRedeemMock).creationCode);
-        vaultBytecodes2[5].vaultKinds.push("ERC7540AsyncRedeemMock");
-        vaultBytecodes2[5].vaultBytecode.push(type(ERC7540AsyncDepositMockRevert).creationCode);
-        vaultBytecodes2[5].vaultKinds.push("ERC7540AsyncDepositMockRevert");
-        vaultBytecodes2[5].vaultBytecode.push(type(ERC7540AsyncRedeemMockRevert).creationCode);
-        vaultBytecodes2[5].vaultKinds.push("ERC7540AsyncRedeemMockRevert");
-        vaultBytecodes2[5].vaultBytecode.push(type(ERC7540AsyncDepositMockRedeemRevert).creationCode);
-        vaultBytecodes2[5].vaultKinds.push("ERC7540AsyncDepositMockRedeemRevert");
+        /// @dev form 3 (7540)
```
Comment (0xTimepunk) [inline test/utils/BaseSetup.sol:2167]: Id 4 here

File: test/utils/ProtocolActions.sol [hunk 5]
```diff
@@ -1430,7 +1430,7 @@ abstract contract ProtocolActions is CommonProtocolActions {
     {
         address vault = IBaseForm(superform).getVaultAddress();
         address asset = IBaseForm(superform).getVaultAsset();
-        if (vault == REAL_VAULT_ADDRESS[SEPOLIA][5]["tUSD"][0] || vault == REAL_VAULT_ADDRESS[ETH][5]["USDC"][0]) {
+        if (vault == REAL_VAULT_ADDRESS[SEPOLIA][3]["tUSD"][0] || vault == REAL_VAULT_ADDRESS[ETH][3]["USDC"][0]) {
```
Comment (0xTimepunk) [inline test/utils/ProtocolActions.sol:1433]: Id 4 here

**Review by 0xTimepunk — COMMENTED** [review]
Should just change 5115 to id 3 and 7540 to id 4 everywhere.

**Comment by sujithsomraaj** [comment]
> Should just change 5115 to id 3 and 7540 to id 4 everywhere.

Done

**Pull request merged**